aws_lambda_builders/workflows/java_gradle/gradle_validator.py

```diff
@@ -83,5 +83,5 @@ def _get_jvm_string(self, gradle_path):
 
         for line in stdout.splitlines():
             l_dec = decode(line)
-            if l_dec.startswith("JVM"):
+            if l_dec.startswith("JVM") or l_dec.startswith("Launcher JVM"):
```

Comment:
nit: This could be simplified to `l_dec.contains("JVM")`

Reply:
I see, I will change it with `.contains(...)` 👍

Reply:
Updated with `in` operator to check the JVM substring.

Reply:
Hi @hnnasit , is there something that I need to do so this PR can be merged? Thank you 🙇

Reply:
Hey @jeffryang24, our CI checks are failing consistently (and randomly out of the blue). The failures are unrelated to your changes, but we're working on fixing them. At the moment, there is no action needed from your side. Thanks for waiting

Reply:
I see, noted 👍